examples/cxx/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright: 2020 Fondazione Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the MIT license
+
+cmake_minimum_required(VERSION 3.16)
+
+project(iDynTreeExamples)
```

Comment:
Isn't this a duplicate of the parent folder?

Reply:
Yep, as a convenience I would like to be able to run cmake from both the examples directory and all the subdirectory. In any case I changed the project name at least.